sale_partner_incoterm/models/sale_order.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Opener B.V. (<https://opener.am>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    incoterm_address_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Incoterm Address",
+    )
+
+    @api.onchange("partner_id")
+    def onchange_partner_id(self):
+        self.update(
+            {
+                "incoterm": self.partner_id.sale_incoterm_id.id,
+                "incoterm_address_id": self.partner_id.sale_incoterm_address_id.id,
+            }
+        )
```

Comment:
```suggestion
    @api.onchange("partner_id")
    def _onchange_partner_id_set_incoterm(self):
        self.update(
            {
                "incoterm": self.partner_id.sale_incoterm_id.id,
                "incoterm_address_id": self.partner_id.sale_incoterm_address_id.id,
            }
        )
```

Alternatively, to not call super and to avoid conflicts with other modules, I find this approach more "secure".
This way it's pretty much impossible to collide.

Reply:
Thanks @ivantodorovich for the review. Your suggestion has been applied.